llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -586,15 +585,15 @@ template<> Type *SPIRVToLLVM::transTypeWithOpcode<spv::OpTypeArray>(
 
     SPIRVWord arrayStride = 0;
     const bool hasArrayStride = pSpvType->hasDecorate(DecorationArrayStride, 0, &arrayStride);
-    LLPC_ASSERT(hasArrayStride ^ (arrayStride == 0));
+    assert(hasArrayStride ^ (arrayStride == 0));
```

Comment:
I think it'd be better to use `!=` here. `^` is an arithmetic, not a logical operator.

Reply:
Again, orthogonal.